data/NotificationsApi.js

```diff
@@ -32,32 +32,39 @@ class NotificationsApi {
   notificationMessage(blueprint, message) {
     let notification = {};
     switch (message) {
-      case 'creating': {
+      case 'imageWaiting': {
         notification = {
-          type: 'process',
+          type: 'info',
+          message: <span>
+            <FormattedMessage
+              defaultMessage="{blueprint} Image creation has been added to the queue."
+              values={{
+                blueprint: <strong>{blueprint}:</strong>
+              }}
+            />
+          </span>,
+          dismiss: true,
+          fade: true,
+        };
+        break;
+      }
+      case 'imageStarted': {
+        notification = {
+          type: 'success',
           message: <span>
             <FormattedMessage
-              defaultMessage="{blueprint} Creating image."
+              defaultMessage="{blueprint} Image creation has started."
```

Comment:
One failed case is due to this change. Just update the 22nd line of `pages/toastNotif.js`.
The ```this.varStatusCreating = `${this.varBlueprintName} Creating image.`;``` should be updated to ```this.varStatusCreating = `${this.varBlueprintName} Image creation has started.`;```